recipes/pyxem/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyXem" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/pyxem/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f95abf7a1f1353eeedb1d616c1083c05cbca3178d13441e1890d41d26864cd3f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py2k]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - hyperspy
+    - transforms3d
+    - scikit-learn
+    - diffpy.structure
+    - spglib
+    - traits
+
+test:
+  requires:
+    - pytest
+  source_files:
+    - tests
+  imports:
+    - pyxem
+    - pyxem.utils
+    - pyxem.io_plugins
+    - pyxem.components
+    - pyxem.generators
+    - pyxem.signals
+    - pyxem.libraries
+  commands:
+    - py.test
```

Comment:
Usually, conda-forge doesn't run the test suite, and I guess the failure on macos could be ignored in the short term, particularly, if this recipe is noarch.
This has been reported in #353, so this should be fine.

Reply:
I disagree that this failure should be disregarded as it shows something is clearly going wrong, apparently not just on macOS, but also in some cases on Windows [as well](https://github.com/pyxem/pyxem/issues/353#issuecomment-472844692).

Reply:
Yes, this is correct: two tests are not behaving well (see https://github.com/pyxem/pyxem/pull/355) and it looks to me that the way pyxem is currently installed with mixing pypi wheel and conda package is a source of inconstancy across different environment/platform.
In that sense, this recipe would be useful to sort out the failing test.
I meant this is not a big deal, because this is not much to do in this recipe, other than patching pyxem, which sounds a bit overkill.
Having a conda-forge package (even with one or two functions broken) would definitely be improvement over the current situation.

Reply:
Use `pytest` instead of `py.test` since this is the recommended way to run it since `pytest 3.0.0`.
https://docs.pytest.org/en/latest/changelog.html#id722
https://github.com/pytest-dev/pytest/issues/1629